contracts/exchange/test/reference_functions.ts

```diff
@@ -107,23 +107,6 @@ describe('Reference functions', () => {
             ).to.throw(expectedError.message);
         });
 
-        it('reverts if `order.makerAssetAmount` is 0', () => {
```

Comment:
this is actually not the correct behavior for the reference function, since the `makerAssetFilledAmount` is computed as `makerAssetAmount * takerAssetFilledAmount / takerAssetAmount`